sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -45,16 +47,18 @@ CosmosDiagnostics clientSideRequestStatistics(ClientSideRequestStatistics client
      */
     @Override
     public String toString() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("userAgent=").append(USER_AGENT).append("\n");
```

Comment:
Is there an Environment.NewLine or equivalent for Java? It might not work correctly on Windows where new line is `\r\n`

Reply:
Good point @j82w, we can use - `System.lineSeparator()`
https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--

I will port that change as a separate PR for everywhere in SDK code.

Reply:
Tracking this issue here : https://github.com/Azure/azure-sdk-for-java/issues/11594